modules/navier_stokes/doc/content/source/variables/BernoulliPressureVariable.md

```diff
@@ -29,11 +29,28 @@ dimension explicit inlet boundary conditions related to velocity, one explicit
 inlet boundary condition related to temperature, and one explicit outlet
 boundary condition related to pressure. Thus physics-based discretizations of
 Euler flows typically (at least partially) upwinds information being advected by
-the flow field. However, given the progagation of pressure information upstream
+the flow field. However, given the propagation of pressure information upstream
 from the explicit outlet pressure boundary condition, we believe it reasonable
 to do the same for computing the Bernoulli pressure jump at porosity
 discontinuities.
 
+!alert! note title=Behavior with parallel execution
+
+In certain cases multiple porosity jump faces may be connected by several cells in a
+chain. For example at corners or when porous medium zones are one-cell wide. In such
+scenarios, the two-term expansion in the downstream cell for the determination of the
+face pressure can lead to a very long chain of calls, which can potentially lead to
```

Comment:
let s not talk about chains of calls.
Maybe require an extended stencil in the direction of the porosity jumps?